crates/sui-indexer/src/pg_integration.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::errors::IndexerError;
+use crate::store::PgIndexerStore;
+use crate::{new_pg_connection_pool, Indexer, IndexerConfig, PgPoolConnection};
+use diesel::migration::MigrationSource;
+use diesel_migrations::{embed_migrations, EmbeddedMigrations, MigrationHarness};
+use prometheus::Registry;
+use tokio::task::JoinHandle;
+
+const MIGRATIONS: EmbeddedMigrations = embed_migrations!("migrations");
+
+pub fn reset_database(conn: &mut PgPoolConnection) {
+    conn.revert_all_migrations(MIGRATIONS).unwrap();
+    conn.run_migrations(&MIGRATIONS.migrations().unwrap())
+        .unwrap();
+}
+
+pub async fn start_indexer(
```

Comment:
can you add a comment of this function sth like:
this function is to spawn indexer thread for local dev with provided Postgres DB url?